data/successors/successor jobs.txt

```diff
@@ -0,0 +1,788 @@
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 2000 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: High Value Delivery Out"
+	job
+	repeat
+	name "High-value shipment to <planet>"
+	description "A courier is needed to transport <cargo> from <origin> to <planet>. You will be paid <payment> on delivery."
+	cargo "Successor High Value" 6 9 .2
+	to offer
+		has "known to the successors"
+		random < 10
+	source
+		attributes "successor"
+		not attributes "crime"
+	destination
+		attributes "successor"
+		attributes "wealthy"
+		not attributes "crime" "uninhabited"
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 3000 400
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: Aid Delivery Out 1"
+	job
+	repeat
+	name "Aid shipment to <planet>"
+	description "A courier is needed to deliver <tons> of aid supplies from <origin> to <planet> by <date>. You will be paid <payment> on delivery."
+	deadline
+	deadline 2
+	cargo random 5 10 .2
+	to offer
+		has "known to the successors"
+		random < 10
+	source
+		attributes "successor"
+		not attributes "crime"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		attributes "crime"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 2400 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: Large Delivery Out 1"
+	job
+	repeat
+	name "Bulk shipment to <planet>"
+	description "A courier is needed for <tons> of supplies from <origin> to <planet>. You will be paid <payment> on delivery."
+	cargo random 30 80 .2
+	to offer
+		has "known to the successors"
+		random < 20
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 1900 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: Large Delivery Out 2"
+	job
+	repeat
+	name "Bulk shipment to <planet>"
+	description "A courier is needed for <tons> of supplies from <origin> to <planet>. You will be paid <payment> on delivery."
+	cargo random 50 70 .2
+	to offer
+		has "known to the successors"
+		random < 20
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 1900 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: Rush Delivery Out 1"
+	job
+	repeat
+	name "Rush shipment to <planet>"
+	description "Due to a logistical error, <tons> of supplies were not carried on the last shipment and need to get from <origin> to <planet> by <date>. You will be paid <payment> on delivery."
```

Comment:
The "last shipment" part is kind of confusing here. Presumably, unless the destination is a station, there are multiple entities and many shipments providing supplies?